test/tests.py

```diff
@@ -548,6 +548,16 @@ class Foo(JsonObject):
         self.assertIsInstance(foo.bar[0], Bar)
         self.assertIsInstance(foo.to_json()['bar'][0], dict)
 
+    def test_list_plus_equals(self):
+        class Foo(JsonObject):
+            bar = ListProperty()
+
+        foo = Foo()
+        foo.bar = [1, 2, 3]
+        self.assertEqual(foo.bar, foo.to_json()['bar'])
+        foo.bar += [4]
+        self.assertEqual(foo.bar, foo.to_json()['bar'])
```

Comment:
Since the very intuitiveness of assignment and such is in question, I would change the asserts to explicitly check the value with
```
self.assertEqual(foo.bar, [1, 2, 3])
self.assertEqual(foo.to_json()['bar'], [1, 2, 3])
```
and
```
self.assertEqual(foo.bar, [1, 2, 3, 4])
self.assertEqual(foo.to_json()['bar'], [1, 2, 3, 4])
```